make/modules/jdk.sctp/Java.gmk

```diff
@@ -47,10 +47,24 @@ ifeq ($(call isTargetOs, aix), true)
   EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
 endif
 
+ifeq ($(OPENJDK_TARGET_OS_ENV), bsd.openbsd)
+  EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
+  ## WORKAROUND need to provide Impl classes that throw UnsupportedOperationException
+  SRC_SUBDIRS += macosx/classes
+endif
+
+ifeq ($(OPENJDK_TARGET_OS_ENV), bsd.netbsd)
+  EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
+  ## WORKAROUND need to provide Impl classes that throw UnsupportedOperationException
```

Comment:
I'm not sure what is meant by "WORKAROUND" here, but there is no `macosx/classes` directory to include, so this accomplishes nothing. I guess this is an older kludge that is no longer needed, since the code below will make sure the UnsupportedUtil.java class is included.